lib/exception_handling/methods.rb

```diff
@@ -16,7 +16,9 @@ def log_error_rack(exception_or_string, exception_context = '', rack_filter = ''
     end
 
     def log_warning(message)
-      log_error(Warning.new(message))
+      warning = Warning.new(message)
+      warning.set_backtrace([])
+      log_error(warning)
```

Comment:
I had actually started with a test in that one as well, but made the mistake of removing it when it wouldn't comply (and when I had noticed that none of the other methods were being tested). I can add it back in and get it to work